### ISSUE TITLE
RFC: 3.0 time locale configuration

### ISSUE THREAD

**Issue by thebigredgeek** [visionmedia/debug#402]
discuss!

**Comment by fab1an** [comment]
Hi, i have come to this thread, because I noticed that time-stamps were off in my logs.

Speaking from many years of experience, I wouldn't use any built-in Date primitives. 
The most sensible solution for doing anything date-related is using the java8-time API. There is a port for javascript https://js-joda.github.io/js-joda
(Yes, I've used `moment` in the past)

To prove my point, here is how js-joda does everything right without any configuration:
```
const {ZonedDateTime} = require("js-joda").use(require("js-joda-timezone"))
const debug = require("debug")("")
debug.useColors = false
debug("<--- NODE | JS-JODA --->   ", ZonedDateTime.now().toString())
```

Output:
```
Sat, 05 Aug 2017 06:48:28 GMT  <-- NODE | JS-JODA --> 2017-08-05T08:48:28.541+02:00[Europe/Vienna]
```

js-joda got the correct timezone for my system, which is everything I want for a logging-library.
